tests/run_tests.sh

```diff
@@ -26,44 +61,55 @@ for PYTHON in /opt/python/*/bin/python; do
 	$PYTHON $MY_DIR/ssl-check.py
 	IMPLEMENTATION=$(${PYTHON} -c "import sys; print(sys.implementation.name)")
 	PYVERS=$(${PYTHON} -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
-	if [ "${IMPLEMENTATION}" == "pypy" ]; then
-		LINK_PREFIX=pypy
-	else
-		LINK_PREFIX=python
+	if [ "${IMPLEMENTATION}" == "cpython" ]; then
 		# Make sure sqlite3 module can be loaded properly and is the manylinux version one
 		# c.f. https://github.com/pypa/manylinux/issues/1030
 		$PYTHON -c 'import sqlite3; print(sqlite3.sqlite_version); assert sqlite3.sqlite_version_info[0:2] >= (3, 34)'
 		# Make sure tkinter module can be loaded properly
 		$PYTHON -c 'import tkinter; print(tkinter.TkVersion); assert tkinter.TkVersion >= 8.6'
+		# cpython shall be available as python
+		LINK_VERSION=$(python${PYVERS} -VV)
+		REAL_VERSION=$(${PYTHON} -VV)
+		test "${LINK_VERSION}" = "${REAL_VERSION}"
 	fi
-	# pythonX.Y / pypyX.Y shall be available directly in PATH
-	LINK_VERSION=$(${LINK_PREFIX}${PYVERS} -V)
-	REAL_VERSION=$(${PYTHON} -V)
+	# cpythonX.Y / pypyX.Y shall be available directly in PATH
+	LINK_VERSION=$(${IMPLEMENTATION}${PYVERS} -VV)
+	REAL_VERSION=$(${PYTHON} -VV)
 	test "${LINK_VERSION}" = "${REAL_VERSION}"
 
 	# check a simple project can be built
-	SRC_DIR=/tmp/forty-two-${IMPLEMENTATION}${PYVERS}
-	DIST_DIR=/tmp/dist-${IMPLEMENTATION}${PYVERS}
-	cp -rf ${MY_DIR}/forty-two ${SRC_DIR}
-	PY_ABI_TAGS=$(basename $(dirname $(dirname $PYTHON)))
-	EXPECTED_WHEEL_NAME=forty_two-0.1.0-${PY_ABI_TAGS}-linux_${AUDITWHEEL_ARCH}.whl
-	${PYTHON} -m build -w -o ${DIST_DIR} ${SRC_DIR}
-	if [ ! -f ${DIST_DIR}/${EXPECTED_WHEEL_NAME} ]; then
-		echo "unexcpected wheel built: '$(basename $(find ${DIST_DIR} -name '*.whl'))' instead  of '${EXPECTED_WHEEL_NAME}'"
-		exit 1
-	fi
-	auditwheel repair --only-plat -w ${DIST_DIR} ${DIST_DIR}/${EXPECTED_WHEEL_NAME}
-	REPAIRED_WHEEL=$(find ${DIST_DIR} -name "forty_two-0.1.0-${PY_ABI_TAGS}-*${AUDITWHEEL_POLICY}_${AUDITWHEEL_ARCH}*.whl")
-	if [ ! -f "${REPAIRED_WHEEL}" ]; then
-		echo "invalid repaired wheel name"
-		exit 1
-	fi
-	${PYTHON} -m pip install ${REPAIRED_WHEEL}
-	if [ "$(${PYTHON} -c 'import forty_two; print(forty_two.answer())')" != "42" ]; then
-		echo "invalid answer, expecting 42"
-		exit 1
+	if [ "${IMPLEMENTATION}" != "graalpy" ]; then
+		SRC_DIR=/tmp/forty-two-${IMPLEMENTATION}${PYVERS}
+		DIST_DIR=/tmp/dist-${IMPLEMENTATION}${PYVERS}
+		cp -rf ${MY_DIR}/forty-two ${SRC_DIR}
+		PY_ABI_TAGS=$(basename $(dirname $(dirname $PYTHON)))
+		EXPECTED_WHEEL_NAME=forty_two-0.1.0-${PY_ABI_TAGS}-linux_${AUDITWHEEL_ARCH}.whl
+		${PYTHON} -m build -w -o ${DIST_DIR} ${SRC_DIR}
+		if [ ! -f ${DIST_DIR}/${EXPECTED_WHEEL_NAME} ]; then
```

Comment:
On x86_64, the wheel does not have the expected name. The difference in behavior with aarch64 comes from the 32bit check of wheel only implemented for x86_64 but not aarch64

Reply:
marking as resolved in this PR. follow up in https://github.com/pypa/manylinux/pull/1520